### DISCUSSION TITLE
[Help]: How to disable drag?

### DISCUSSION THREAD

**Discussion by ngocducpham** [davidjerleke/embla-carousel#635] [Help] [answered]
Hello, I'm using embla carousel for my web project and I want to disable the drag and drop functionality of the carousel. I only want to use the navigation buttons to change the slides. Is there any option or method to achieve this?
This is sandbox: https://codesandbox.io/s/embla-carousel-arrow-dots-react-forked-csdrdf

**Comment by davidjerleke** [comment, marked as answer]
Hi @ngocducpham,

Sure. Set the [`watchDrag`](https://www.embla-carousel.com/api/options/#watchdrag) option to `false`.

Best,
David

**Comment by gnllucena** [comment]
@davidjerleke sliding with arrow keys still happens, is there another option to disable that as well?

**Reply by davidjerleke** [reply]
@gnllucena please create a separate discussion for your question.
